reference/core/ingress-controller.md

```diff
@@ -200,7 +200,7 @@ spec:
 
 ```yaml
 ---
-apiVersion: extensions/v1beta1
+apiVersion: extensions/v1
```

Comment:
This sample is out of date with the `v1` API.
@alexgervais to provide a good and valid example.